src/chains/ethereum/utils/package-lock.json

```diff
@@ -1,2156 +1,2156 @@
 {
-	"name": "@ganache/ethereum-utils",
```

Comment:
Is this file _just_ whitespace changes? It's real hard to tell.

Reply:
If you want to confirm that the package-locks are inline with the package.json, you can always `npm run reinstall` locally to confirm that there are no diffs. Other than that, I don't know what we could do to confirm the validity of these files.

Reply:
I think you need to restore the original whitespace. I think what happened here is that your version of npm has different rules for which whitespace to use in a package-lock. I think some versions of npm preserve what was there before, but others like to change it _sometimes_.

This has happened to me a few times and what I've done in the past is disabled prettier (just in case it is doing weird things), then did a find/replace for `<new whitespace>`/`<old whitespace>` individually within each package-lock npm screwed up. If using VM code you have to do this many many times, since it only will find/replace a few hundred matches at a time.

Reply:
ahhhh got ya. Looks like that worked, thanks @davidmurdoch!